### DISCUSSION TITLE
issues with Japanese IME cannot right click

### DISCUSSION THREAD

**Discussion by glowingwire** [valinet/ExplorerPatcher#1659] [General]
I am setting up a windows 11 computer for a family member, and your product is vital. 

I am installing the Japanese IME.  It instructs me to right click on the A to switch between the 3 types of writing.

it does not work.   Windows 11 is garbage, but the fact that the taskbar is crashing at the same time as that not working lets me think this is an issue

as of today, I have the latest build with the "get latest updates" updates.

**Comment by pyrates999** [comment]
Please post windows build.  You can find this out by running winver.
Please post explorer patcher version.

**Comment by glowingwire** [comment]
22621.1992

I am also running openshell latest release

explorer patcher:
22621.1555.55.1

I am running taskbar on the left, but also tried it on thr bottom.

**Comment by pyrates999** [comment]
Try the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher
7. reboot

Then apply your settings again in explorer patcher to see if it fixes it.

**Reply by vippium** [reply]
For Ease, you can uninstall EP with iobit. It will automatically delete all registry keys and all residual files too. No need to do this manually
